modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/css/main.scss

```diff
@@ -8,4 +8,8 @@
 			width: 32px;
 		}
 	}
+
+	.table tbody tr {
```

Comment:
@andrea-ale-sbarra , I will check and take care of it.

Reply:
thank you!

Reply:
@andrea-ale-sbarra , I took a look at this issue. This will only affect the specifications table because the css only targets .portlet-commerce-product-content which is our product details portlet. Let me know if this answer is satisfactory.

Reply:
Ehi @brianikim then it's fine thank you!